internal/compose/compose.go

```diff
@@ -214,6 +216,11 @@ func NewProject(name string, paths ...string) (*Project, error) {
 		c.disablePullProgressInformation = true
 	}
 
+	v, ok = os.LookupEnv(DisableProgressOutputComposeEnv)
+	if !c.dockerComposeV1 && !c.dockerComposeStandalone && ok && strings.ToLower(v) != "false" {
```

Comment:
This option is just available using `docker compose`

Reply:
Why? This behaves the same in `docker-compose` (trying with standalone docker-compose `v2.23.3`).

Reply:
I was trying with `docker-compose` with version 2.17.2 (same version that was set previously in CI for it). In that version there is no `--progress` flag.

I see that in the latest version of docker-compose standalone (v2.24.1), that version is already in place.